spec/features/create_mediated_page_request_spec.rb

```diff
@@ -7,9 +7,11 @@
       visit new_mediated_page_path(item_id: '1234', origin: 'SPEC-COLL', origin_location: 'STACKS')
       click_link "I don't have a SUNet ID"
 
+      expect(page).to have_field('Library ID', type: 'text')
       expect(page).to have_field('Name', type: 'text')
       expect(page).to have_field('Email', type: 'email')
       expect(page).to have_css('a', text: '‹ Go back (show the login option)')
+      expect(page).to have_css('input#mediated_page_user_attributes_library_id', :focus)
```

Comment:
Okay, so this is a bit hacky but I believe should work.  Replace that expectation (and the others like it) with:

```ruby
expect(page).to have_css('input#mediated_page_user_attributes_library_id')
expect(page.evaluate_script('document.activeElement.id')).to eq 'mediated_page_user_attributes_library_id'
```